app/utils/answersHelpers/HolderOfTheTransitProcedureHelper.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.answersHelpers
+
+import generated.AddressType10
+import models.reference.Country
+import models.{DynamicAddress, UserAnswers}
+import pages.holderOfTheTransitProcedure.CountryPage
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewModels.sections.Section
+import viewModels.sections.Section.AccordionSection
+
+class HolderOfTheTransitProcedureHelper(
+  userAnswers: UserAnswers
+)(implicit messages: Messages)
+    extends AnswersHelper(userAnswers) {
+
+  def identificationNumber: Option[SummaryListRow] = buildRowWithNoChangeLink[String](
+    data = userAnswers.ie043Data.HolderOfTheTransitProcedure.flatMap(_.identificationNumber),
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.rowHeadings.holderOfTheTransitProcedure.identificationNumber"
+  )
+
+  def tirHolderIdentificationNumber: Option[SummaryListRow] = buildRowWithNoChangeLink[String](
+    data = userAnswers.ie043Data.HolderOfTheTransitProcedure.flatMap(_.TIRHolderIdentificationNumber),
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.rowHeadings.holderOfTheTransitProcedure.tirHolderIdentificationNumber"
+  )
+
+  def name: Option[SummaryListRow] = buildRowWithNoChangeLink[String](
+    data = userAnswers.ie043Data.HolderOfTheTransitProcedure.map(_.name),
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.rowHeadings.holderOfTheTransitProcedure.name"
+  )
+
+  def address: Option[SummaryListRow] = buildRowWithNoChangeLink[DynamicAddress](
+    data = userAnswers.ie043Data.HolderOfTheTransitProcedure.map(_.Address.toDynamicAddress),
+    formatAnswer = formatAsDynamicAddress,
+    prefix = "unloadingFindings.rowHeadings.holderOfTheTransitProcedure.address"
+  )
+
+  def country: Option[SummaryListRow] = buildRowWithNoChangeLink[Country](
+    data = userAnswers.get(CountryPage),
+    formatAnswer = formatAsCountry,
+    prefix = "unloadingFindings.rowHeadings.holderOfTheTransitProcedure.country"
+  )
+
+  def holderOfTheTransitProcedureSection: Seq[Section] = userAnswers.ie043Data.HolderOfTheTransitProcedure
+    .map(
+      _ =>
```

Comment:
Very minor but could utilise this and pass the various values into the methods, just so we're not having to call `userAnswers.ie043Data.HolderOfTheTransitProcedure` over and over again